src/components/list/list-header.ts

```diff
@@ -1,6 +1,13 @@
 import { html, LitElement } from 'lit';
 import { styles } from './list-header.material.css';
 
+/**
+ * Header component for list items.
```

Comment:
Isn't this just a header list item. "Header component for list items" kind of has a different meaning.